app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -145,13 +145,19 @@ def remove_from_environment
     param :system_environment_id, :number, :desc => N_("environment to reassign orphaned systems to")
     param :key_content_view_id, :number, :desc => N_("content view to reassign orphaned activation keys to")
     param :key_environment_id, :number, :desc => N_("environment to reassign orphaned activation keys to")
+    param :force_destory_cv, :boolean, :desc => N_("delete the content view if all versions are also removed")
```

Comment:
i would probably not consider this a 'force', its more of a convenience method to delete after removal.  Maybe just rename it to 'destroy_content_view' ?